Resolve feedback for chapter 11 (#975)
browserengineering/book: book/visual-effects.md

image blended with the white background. Doing the operations in a
different order would lead to dark-orange or black text.

To handle this properly, browsers apply blending not to individual shapes but to
a tree of surfaces. Conceptually, each surface is drawn individually,
and then blended into its parent surface. Rastering a web page requires a
bottom-up traversal of the tree: to raster a surface you first need to raster
its contents, including its child stacking contexts, and then the whole
contents need to be blended together into the parent.[^stacking-context-disc]

[^stacking-context-disc]: This tree of surfaces is an implementation strategy
and not something required by any specific web API. However, the web does
define the concept of a [*stacking context*][stacking-context], which is
related. A stacking context is technically a mechanism to define groups and

> ordering during paint, and stacking contexts need not come with a surface

Reviewer: ```suggestion
ordering during paint, and stacking contexts need not correspond to a surface
```
